sample-app/build.gradle

```diff
@@ -89,7 +89,7 @@ dependencies {
     implementation 'com.squareup.okhttp3:logging-interceptor:4.10.0'
     implementation 'com.github.skydoves:sandwich:1.3.2'
     implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.4'
-
+    implementation 'io.sentry:sentry-android:6.25.2'
```

Comment:
You don't need to add sentry to the sample app! It will be injected by the Forage SDK when that is installed :)

Reply:
f5a9955925fbf82977bf3c44016df03e4dd2ce9c